### PR TITLE
feat(stats): add BME dashboard page

### PR DESCRIPTION
**Pull request by baktun14**
https://github.com/user-attachments/assets/8ad544e8-3c85-43ca-8de4-70d6124de577

## Why

Closes CON-70

Adds a dedicated BME (Burn-Mint Equilibrium) dashboard to the stats web app, providing visibility into AKT/ACT burn and mint mechanics, vault health, and circuit breaker status.

## What

- New BME dashboard page at `/bme` with summary cards, AKT burn/ACT mint metrics, ACT burn/AKT remint metrics, collateral ratio, and circuit breaker status
- BME graph pages for per-metric time series visualization
- Backend API endpoints for BME dashboard data and status history
- BME indexer for tracking on-chain burn/mint events, status changes, and vault seeding
- Database models and migrations for BME event storage
- Single consolidated dashboard endpoint with period comparison (now vs 24h ago)
- CoinGecko AKT price fallback for collateral ratio calculation
- Real-time percentage change indicators on all metric cards

## Summary by CodeRabbit

* **New Features**
  * BME Analytics Dashboard page with KPI cards, comparisons, collateral ratio and circuit-breaker status
  * BME metric graph pages with time-range selection, CSV export and per-snapshot pages
  * New BME API endpoint for dashboard data; client hooks, query keys and URL helpers to fetch dashboard & status history
  * Navigation links added for BME (desktop & mobile)

* **Bug Fixes / Improvements**
  * Tooltip date handling adjusted for consistent UTC display
  * Collateral ratio and snapshot retrieval improvements for more accurate/robust values

* **Tests**
  * Expanded tests covering BME calculations, edge cases and snapshot behavior

### CODE REVIEW

File: apps/api/src/dashboard/services/stats/stats.service.ts [hunk 1]
```diff
@@ -237,13 +310,80 @@ export class StatsService {
...
   }
 
+  @Memoize({ ttlInSeconds: minutesToSeconds(5) })
+  async getBmeDashboardData(): Promise<BmeDashboardDataResponse> {
+    const { now, compare, secondCompare } = await this.statsRepository.findBmeDashboardData();
+
+    let collateralRatio = now.collateralRatio;
+
+    if (collateralRatio === 0 && now.outstandingUact > 0 && now.vaultUakt > 0) {
+      const aktPrice = await this.getAktPrice();
```
Comment (stalniy) [inline apps/api/src/dashboard/services/stats/stats.service.ts:335]: **thought:** akt price should be fetched from blockchain:

```ts
  const response = await sdk.akash.oracle.v1.getAggregatedPrice({
    denom: "akt",
  });

{
  aggregatedPrice: {
    denom: 'akt',
    twap: '0.506542162000000000',
    medianPrice: '0.507108910000000000',
    minPrice: '0.507108910000000000',
    maxPrice: '0.507108910000000000',
    timestamp: 2026-03-19T21:10:36.236Z,
    numSources: 1,
    deviationBps: Long { low: 0, high: 0, unsigned: false }
  },
  priceHealth: {
    denom: 'akt',
    isHealthy: true,
    hasMinSources: true,
    deviationOk: true,
    totalSources: 2,
    totalHealthySources: 1,
    failureReason: []
  }
}
```

Comment (baktun14) [reply apps/api/src/dashboard/services/stats/stats.service.ts:335]: Yea now that we have the oracle we should replace the coingecko fetching or prices

Comment (baktun14) [reply apps/api/src/dashboard/services/stats/stats.service.ts:335]: create an issue for (CON-105)

**Review by stalniy — APPROVED** [review]

File: apps/stats-web/src/app/bme-graph/[snapshot]/GraphContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+export interface IGraphProps {
+  snapshot: BmeSnapshots | SNAPSHOT_NOT_FOUND;
+}
+
+export default function GraphContainer({ snapshot }: IGraphProps) {
+  const [selectedRange, setSelectedRange] = useState(SELECTED_RANGE_VALUES["7D"]);
+  const { data: snapshotData, status } = useGraphSnapshot(snapshot);
+  const snapshotMetadata = snapshotData && getBmeSnapshotMetadata(snapshot as BmeSnapshots);
+  const rangedData = snapshotData && snapshotData.snapshots.slice(Math.max(snapshotData.snapshots.length - selectedRange, 0), snapshotData.snapshots.length);
```
[the remark this answers is not in the document]
Comment (baktun14) [reply apps/stats-web/src/app/bme-graph/[snapshot]/GraphContainer.tsx:31]: Fixed in 62eba5a — added explicit fallback UI for `NOT_FOUND` and `error` states, and guarded the spinner behind `snapshot \!== NOT_FOUND`.
